### PR TITLE
[DVCSMP-3471] Update zigbee fingerprints for Sengled

### PR DESCRIPTION
**Pull request by munds**
This PR updates zigbee fingerprint so that the device join name is not misreported.

### CODE REVIEW

**Comment by workingmonk** [comment]
@tpmanley will make the call on whether to merge it to staging or master

**Review by tpmanley — DISMISSED** [review]
Looks good. I tested with a BR30 that was previously being fingerprinted as the A19. I feel this is safe to go out tomorrow so can be rebased on staging.

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 1]
```diff
@@ -30,8 +30,13 @@ metadata {
...
-        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E12-N14", deviceJoinName: "Sengled Element Classic BR30"
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E11-G13", deviceJoinName: "Sengled Element Classic"
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E11-G23", deviceJoinName: "Sengled Element Classic"
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E11-G33", deviceJoinName: "Sengled Element Classic"
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E11-G14", deviceJoinName: "Sengled Element Classic"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: Wouldn't hurt to put this right after the E11-G13 fingerprint just so it's alpha-numerically ordered

Comment (munds) [reply devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: Ok! I had it like that before but it was easier in my head to sort the 13/23/33 and then the 14. Maybe my head is wired weird 😛 

Change forthcoming.

**Review by tpmanley — APPROVED** [review]

**Pull request merged**